test/Interop/Cxx/enum/c-enums-withOptions-omit.swift

```diff
@@ -0,0 +1,8 @@
+// RUN: %target-swift-ide-test -print-module -module-to-print=CenumsWithOptionsOmit -I %S/Inputs -source-filename=x -enable-experimental-cxx-interop | %FileCheck %s
```

Comment:
The reason this is failing on windows and linux is because objc interop isn't enabled. Not sure if it's supported on those platforms, so maybe add a requires: objc_interop.

Reply:
See this comment. This is still going to fail on windows/linux.

Reply:
Oh, I missed the requires at the top. Usually that goes under the run.